collectors/ebpf.plugin/integrations/ebpf_processes.md

```diff
@@ -87,7 +87,7 @@ Metrics:
 |:------|:----------|:----|
 | app.process_create | calls | calls/s |
 | app.thread_create | call | calls/s |
-| app.task_exit | callp | calls/s |
```

Comment:
undo this change, the docs gen script will update this file

Reply:
Done!